include/wx/qt/colour.h

```diff
@@ -10,37 +10,35 @@
 #ifndef _WX_QT_COLOUR_H_
 #define _WX_QT_COLOUR_H_
 
-#include <QtGui/QColor>
+class QColor;
 
 class WXDLLIMPEXP_CORE wxColour : public wxColourBase
 {
 public:
     DEFINE_STD_WXCOLOUR_CONSTRUCTORS
-    wxColour(const QColor& color) : m_qtColor(color) {}
+    wxColour(const QColor& color);
 
-    virtual bool IsOk() const { return m_qtColor.isValid(); }
+    virtual bool IsOk() const;
 
-    unsigned char Red() const { return m_qtColor.red();   }
-    unsigned char Green() const { return m_qtColor.green(); }
-    unsigned char Blue()  const { return m_qtColor.blue();  }
-    unsigned char Alpha() const { return m_qtColor.alpha(); }
+    unsigned char Red() const;
+    unsigned char Green() const;
+    unsigned char Blue()  const;
+    unsigned char Alpha() const;
```

Comment:
Looking at the accessors above, I think you could just inline them here to return `red`, `green`, `blue`, `alpha` and `valid` members.
...which should all have `m_` prefix as well.